server/routes/events.js

```diff
@@ -3,7 +3,8 @@ var express = require('express');
 var router = express.Router();
```

Comment:
General comment to keep consistent indentation/newlining

e.g. put .then() on a new line after the query line